bigtable/integration_test.go

```diff
@@ -2105,6 +2131,36 @@ func TestIntegration_AdminBackup(t *testing.T) {
 	if _, err := adminClient.TableInfo(ctx, restoredTable); err != nil {
 		t.Fatalf("Restored TableInfo: %v", err)
 	}
+	// Set backup policy.
+	// Get Etag first from GetIAMPolicy.
+	gotPolicy, err := adminClient.GetIAMPolicy(ctx, cluster, "mybackup")
+	if err != nil {
+		t.Fatalf("GetIAMPolicy: %v", err)
+	}
+	policy := &v1.Policy{}
+	role := "roles/bigtable.viewer"
+	members := []string{"user:user1@test.com"}
+	policy.Version = 1
+	policy.Etag = gotPolicy.GetEtag()
+	policy.Bindings = append(policy.Bindings, &v1.Binding{
+		Role:    role,
+		Members: members,
+	})
+	newPolicy, err := adminClient.SetIAMPolicy(ctx, cluster, "mybackup", policy)
+	if err != nil {
+		t.Fatalf("SetIAMPolicy: %v", err)
+	}
+	if diff := testutil.Diff(newPolicy, policy); diff != "" {
+		t.Fatalf("GetIAMPolicy: got - want +\n%s", diff)
```

Comment:
I got this error with `Etag`:
![image](https://user-images.githubusercontent.com/38891896/99378069-095bf300-28d8-11eb-9702-6aba579151c0.png)
Same with `GetIAMPolicy` method
Deleting a table does not affect the result (PASS or FAIL).